.github/workflows/launchpad.yml

```diff
@@ -2,11 +2,13 @@ name: Deploy to Launchpad
 
 on:
   push:
-    # Only applies to our protected branches
+    # Only applies to our protected branches and tags
     branches:
       - master
       - develop
       - '[0-9]+'
+    tags:
+      - '*'
```

Comment:
Uh nice! That will push to launchpad also when we create a tag for the previously created merge commit, right? :)

Reply:
Yes that's correct.

Reply:
Agh, this doesn't actually work due to https://github.com/github/roadmap/issues/825 .